pkg/cmd/release/create/create.go

```diff
@@ -69,6 +72,47 @@ const (
 	//FlagAliasPackagePrereleaseLegacy = "packagePrerelease"
 )
 
+var packageOverrideLoopHelpText = heredoc.Doc(`
+bold(PACKAGE SELECTION)
+ This screen presents the list of packages used by your project, and the steps
+ which reference them. 
+ If an item is dimmed (gray text) this indicates that the attribute is duplicated.
+ For example if you reference the same package in two steps, the second will be dimmed. 
+
+bold(COMMANDS)
+ Any any point, you can enter one of the following:
+ - green(?) to access this help screen
```

Comment:
This is like the world's shittiest version of markdown, but the "bang-for-buck" we get with a handful of regexes is great.
FYI @domenicsim1 @jbristowe if you want to re-use the prettifyHelp function in other places